### ISSUE TITLE
New rule: Disallow ternary conditions inside template literals

### ISSUE THREAD

**Issue by fsubal** [fsubal/eslint-plugin-classnames#117]
Examples of incorrect code for this rule:

```jsx
<button className={`${ cond && 'bg-blue-300' }`}>Hello</button>;
```

This could lead to unexpected results like  `<button className="false">` .
But currently, this code is not warned nor fixed.

Their should be a rule that auto-fixes this into

```jsx
<button className={classNames(cond && 'bg-blue-300')}>Hello</button>;
```

or maybe

```jsx
<button className={classNames({ 'bg-blue-300': cond })}>Hello</button>;
```

**Comment by fsubal** [comment]
Candidates of the name of this rule:

- `no-ternary-conditions-in-template-literals`
- `no-conditional-template-literals`
